benchmarks/main.go

```diff
@@ -0,0 +1,9 @@
+package main
+
```

Comment:
Can we move the benchmarks to the internal package?

Reply:
Roger that, thanks.